mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -22,8 +22,8 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 0 random
 		ARG 1 min
 		ARG 2 max
-	METHOD method_15345 fastInverseSqrt (D)D
-		ARG 0 value
+	METHOD method_15345 fastInverseSqrtd (D)D
```

Comment:
Are these `d` and `f` suffixes needed? The methods also have different input types, and for example `java.lang.Math` doesn't use type suffixes.

Reply:
IntelliJ warned me that "not all usages could be renamed" when I tried to name both methods the same (using Knit). Though I'm not why...